src/chrome/content/rules/Sparkasse-ansbach.de.xml

```diff
@@ -1,5 +1,15 @@
+<!--
+
+Connection closed:
+
+sparkasse-ansbach.de	
+Fetch error: http://sparkasse-ansbach.de/ => https://sparkasse-ansbach.de/: (7, 'Failed to connect to sparkasse-ansbach.de port 443: Connection refused')
+
+-->
 <ruleset name="Sparkasse-ansbach.de">
+	<target host="sparkasse-ansbach.de" />
 	<target host="www.sparkasse-ansbach.de" />
 
-	<rule from="^http:" to="https:" />
+	<rule from="^http://sparkasse-ansbach\.de/" to="https://www.sparkasse-ansbach.de/" />
+	<rule from="^http://www.sparkasse-ansbach\.de/" to="https://www.sparkasse-ansbach.de/" />
```

Comment:
You can keep the `<rule from="^http:" to="https:" />` here. Otherwise you must escape the dot in `www\.`.

Reply:
`<rule from="^http:" to="https:" />` should be the very last rule. This does the same as `<rule from="^http://www\.sparkasse-ansbach\.de/" to="https://www.sparkasse-ansbach.de/" />`